README.md

```diff
@@ -83,4 +83,14 @@ The ``travis.py`` script try to be clever:
 
 It is possible to build local image for testing with the following command:
 
-    SONARSCANNER_VERSION=3.0.3.778 VERSION=3.0 python travis.py --language sonar --pull-request=true
\ No newline at end of file
+mac :
+
+    SONARSCANNER_VERSION=3.0.3.778 VERSION=3.0 python travis.py --language sonar --pull-request=true
```

Comment:
You mean this does not work on Linux??

Reply:
my bad, it does work. I'll correct it

Reply:
Done. I replaced the sonar example with the java example since it shows more env variables